gtsfm/three_view_estimator.py

```diff
@@ -0,0 +1,110 @@
+"""Estimator which operates on a triplet of images to compute relative pose and verified indices.
```

Comment:
i saw this is placed at ` gtsfm/three_view_estimator.py`. Should it be inside gtsfm/frontend? Is it intended as a verifier or as a matcher? Is this more of an abstract class?

Reply:
This is an alternative to the `TwoViewEstimator` plate. This class reuses the two-view's matcher (e.g. TwoWay matcher/SuperGlue) and performs cycle-consistency check to output matches for triplets.

I will need to implement a trifocal based verifier, and that will go into the frontend folder.